identity/conf/routes

```diff
@@ -43,12 +43,3 @@ GET         /complete-consents                      controllers.editprofile.Edit
 # Redirected to Manage
 GET         /privacy/edit                           controllers.editprofile.EditProfileController.redirectToManageEmailPrefs
 GET         /email-prefs                            controllers.editprofile.EditProfileController.redirectToManageEmailPrefs
```

Comment:
Keeping these two for now because it's possible that Newsletter signup still calls some identity routes to pass consent.